README.md

```diff
@@ -49,12 +52,41 @@ If you have an awesome project using `docsify-katex` and want to share it with o
 
 ## Known issues
 
-Making KaTeX work properly with docsify is a hard work, this repo is just a workaround. The main difficulty is that we cannot know context of the whole markdown file like [yzhang-gh/markdown-it-katex](https://github.com/yzhang-gh/markdown-it-katex) does, so matching complicate <code>`</code>'s and <code>\$</code>'s is not possible. Now we have such known issues:
+Making KaTeX work properly with docsify is a hard work, this repo is just a workaround. ~~The main difficulty is that we cannot know context of the whole markdown file like [yzhang-gh/markdown-it-katex](https://github.com/yzhang-gh/markdown-it-katex) does, so matching complicate <code>`</code>'s and <code>\$</code>'s is not possible. Now we have such known issues:~~
 
-1. Change <code>\\`</code> to:
+~~1. Change <code>\\`</code> to:~~
 
-   ```txt
-   <code>`</code>
-   ```
+    ~~By doing this, your <code>`</code> will not be considered as a start or end of a code block.~~
 
-    By doing this, your <code>`</code> will not be considered as a start or end of a code block.
\ No newline at end of file
+## new Install
+
+```html
+<head>
+  <!-- begin for docsify katex -->
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/katex.min.css"
+    integrity="sha384-bYdxxUwYipFNohQlHt0bjN/LCpueqWz13HufFEV1SUatKs1cm4L6fFgCi1jT643X"
+    crossorigin="anonymous"
+  />
+  <script
+    defer
+    src="https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/katex.min.js"
+    integrity="sha384-Qsn9KnoKISj6dI8g7p1HBlNpVx0I8p1SvlwOldgi3IorMle61nQy4zEahWYtljaz"
+    crossorigin="anonymous"
+  ></script>
+  <!-- the version of marked must be less than 4 -->
+  <script src="https://cdn.jsdelivr.net/npm/marked@3"></script>
```

Comment:
I think docsify has a built-in marked dependency, can we avoid this extra script?

Reply:
I'm afraid it can't because the feature 'extension' of marked was added after the version 2.10. However, the version of marked built in docsify is 1.2.9. What's more, it's a pity that version 4 is not compatible with previous.